src/ipc/ipc4/helper.c

```diff
@@ -771,6 +771,19 @@ const struct comp_driver *ipc4_get_drv(uint8_t *uuid)
 	return drv;
 }
 
+bool ipc4_comp_gateway(struct comp_dev *dev)
+{
+	const struct sof_uuid copier = {0x9ba00c83, 0xca12, 0x4a83, {0x94, 0x3c,
+		0x1f, 0xa2, 0xe8, 0x2f, 0x9d, 0xda}};
+
+	if (!memcmp(dev->drv->uid, &copier, UUID_SIZE)) {
```

Comment:
Can you add a comment to explain this logic.

Reply:
sure